dependencies/pom.xml

```diff
@@ -79,6 +79,11 @@
       <artifactId>httpclient</artifactId>
       <version>4.5.12</version>
     </dependency>
+    <dependency>
+      <groupId>org.relaxng</groupId>
```

Comment:
Can you find a license for this? I don't see one anywhere. I think it's BSD but I don't see a statement about that.